### PR TITLE
Updated doc

### PR DESCRIPTION
**Pull request by artall64**
- Add reference on what is keystore, and where a developer can get it
- Add a note on test ether pitfall
- Add reference on GitHub pages docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,5 +43,10 @@ A demo can be run with
 ```
 npm run demo
 ```
-
 and going to `http://localhost:1234` in the browser.
+
+*NOTE: You should have enough test ether to open channel, default value hardcoded in `profile.json` is 100 eth.*
```
Comment (Dominator008) [inline README.md]: This is inaccurate. The value is specified by the `amountWei` parameter in the `openEthChannel` API.

**Comment by Dominator008** [comment]
Thanks for the improvements!

File: README.md (hunk 1 above)
Comment (artall64) [reply README.md]: Fixed

Comment (Dominator008) [reply README.md]: Sorry. What I meant was that it's not hardcoded in `profile.json`, but passed as the `amountWei` parameter. Are you talking about the example in `demo/demo.js`?

Comment (artall64) [reply README.md]: Fixed, I was in hurry ;-)

**Pull request merged**